.gitignore

```diff
@@ -159,4 +159,5 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-.vscode/*
\ No newline at end of file
+.vscode/*
```

Comment:
@sone2000, please do not include these .vscode files in the PR, they are IDE dependent.